Packs/OpenCVE/Integrations/OpenCVE/OpenCVE.py

```diff
@@ -480,7 +480,17 @@ def parse_cve(ocve: OpenCVE, cve: dict):
                 {'metrics': 'Confidentiality', 'value': ocve._map(cvssV2.get('confidentialityImpact', None))}
             ]
 
-    parsed_cve['fields']['nodes'] = cve.get('raw_nvd_data', {}).get('configurations', {}).get('nodes', {})
+    # Backwards compatibility, the configuration used to be a dict. Now it is a list.
+    configurations = cve.get('raw_nvd_data', {}).get('configurations')
+    nodes = []
+    if isinstance(configurations, dict):
+        nodes = configurations.get('nodes', [])
+    elif isinstance(configurations, list):
+        for config in configurations:
+            nodes.extend(config.get('nodes', []))
+    # Design decision: The nodes are aggregated into a single list, this means we won't support 'AND' Statements in CPE matching,
```

Comment:
The design decision was done with Dror

Reply:
What's `CPE`? Maybe explain it somewhere?

Also, is `raw_nvd_data.configurations` always present?
If you already touch this area, maybe test that they exist?

Reply:
CPE is a known security term, Common Platform Enumeration.

and the configuration might not exist, that's why i'm doing a `get` on it.